packages/@aws-cdk/aws-config/test/integ.custompolicy.ts

```diff
@@ -23,6 +23,14 @@ rule checkcompliance when
 }
 `;
 
+// const samplePolicyText2 = `
+
+// rule username when
+//     resourceType == "AWS::IAM::User" {
+//       configuration.userName == "sample-user"
+//     }
+// `;
+
```

Comment:
I'm assuming that this being commented out is unintentional?

Reply:
Sorry, this is Integ-test, I forgot to turn it off while debugging.

Reply:
fixed